test/events.js

```diff
@@ -1,156 +1,161 @@
-export function testEvents(options, innerElement) {

```

Comment:
I'm wrong or it loos like completely change? Maybe lien separator?

Reply:
It was ndentation level change